introduction.md

```diff
@@ -1 +1,9 @@
 Hello, my name is Filip and my favorite CS path is JavaScript.
+
+#### My deadly skillz are:
+
+* playing guitar
+* photography
+* sales/marketing
+* html/css
+* javascript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?